api/ceramic.yaml

```diff
@@ -156,9 +156,69 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ErrorResponse"
+  "/experimental/events/{sep}/{model}":
+    get:
+      summary: Get events matching the interest stored on the node
+      parameters:
+        - name: sep
+          in: path
+          description: name of the separator/sort_key e.g. 'model'
+          schema:
+            type: string
+          required: true
+        - name: model
+          in: path
+          description: value associated with the sort key e.g. multibase encoded model ID
```

Comment:
The value of the field in the Events header indicated by the separator key.

Reply:
updated